sw/device/lib/testing/hmac_testutils.h

```diff
@@ -11,10 +11,91 @@
 #include "sw/device/lib/dif/dif_hmac.h"
 
 /**
- * Read and compare the length of the message in the HMAC engine to the length
- * of the message sent in bits.
+ * Timeouts to be used for different HMAC operations.
+ *
+ * All timeouts are calculated assuming a 100kHz clock frequency, in order
+ * to cover a range of targets. Please see:
+ * https://docs.opentitan.org/hw/ip/hmac/doc/
```

Comment:
HMAC runs on the same clock frequency as the cpu. so maybe these could be derived using `kClockFreqCpuHz` value instead? That would make the test robust across all platforms.

Reply:
Done.

Also, I think my previous calculations might have been wrong. If you could double check me @sriyerg